test.js

```diff
@@ -183,7 +183,7 @@ test('unist-util-visit', function (t) {
     st.end()
 
     /**
-     * @param {Node} node
+     * @param {Node & { value: string}} node
```

Comment:
this should probably be a Literal https://github.com/DefinitelyTyped/DefinitelyTyped/blob/18043fe2428e3e6bfe5c10d46852befe1996b6dd/types/unist/index.d.ts#L95-L97
```suggestion
     * @param {Literal} node
```

Reply:
Ah yeah - I was going to use `Literal` but the typechecker in my head said that `unknown != string`, oops. Thanks.